modules/govuk/templates/ckan/pycsw.cfg.erb

```diff
@@ -0,0 +1,90 @@
+# =================================================================
+#
+# Authors: Tom Kralidis <tomkralidis@gmail.com>
+#
+# Copyright (c) 2015 Tom Kralidis
+#
+# Permission is hereby granted, free of charge, to any person
+# obtaining a copy of this software and associated documentation
+# files (the "Software"), to deal in the Software without
+# restriction, including without limitation the rights to use,
+# copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following
+# conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# =================================================================
+
+[server]
+home=/var/ckan
+url=<%= @ckan_site_url%>/csw
+mimetype=application/xml; charset=UTF-8
+encoding=UTF-8
+language=en-US
+maxrecords=10
+loglevel=DEBUG
+logfile=/var/log/ckan/pycsw.log
```

Comment:
Does log rotation need configuring for this log file? Also, the `DEBUG` log level sounds like it might be a little verbose?

Reply:
Not sure if log rotation is supported, could downgrade the level to `INFO` but for initial release it might be worth leaving it at `DEBUG` to resolve issues.

Reply:
We'll get that logrotate configuration for free:

```
thomasleese@ec2-integration-blue-ckan-ip-10-1-4-114:/etc/logrotate.d$ cat govuk-ckan
/var/log/ckan/*.log {
  daily
...
```

Reply:
:+1: